samples/ingress/frontend/components/TodoFooter.js

```diff
@@ -11,7 +11,7 @@ export default class TodoFooter extends React.Component {
         return (
             <div className="filters">
                 <div className="todo-count">{this.props.todoCount} {activeTodoWord} left</div>
-                <div className="stats-link"><Link prefetch href="/stats"><a>stats</a></Link></div>
+                <div className="stats-link"><Link prefetch href="/stats" legacyBehavior><a>stats</a></Link></div>
```

Comment:
@cxznmhdcxz Ruijun can you add the reason behind adding this ```legacyBehavior``` tag to this div please ?

Reply:
@hsubramanianaks  According to Next.js docs, starting with  version 13 using `<a>` as child of `<Link>` is invalid, and adding this could render the page as expected again. Tested and it works.